runtime.js

```diff
@@ -121,8 +119,15 @@
         } else {
           Promise.resolve(info.value).then(callNext, callThrow);
         }
+      };
+
+      if (process && process.domain) {
```

Comment:
This will still throw in browsers since `process` wont be defined. You'll need to do `typeof process !== "undefined"`.